extra/docs_landing.html

```diff
@@ -48,335 +48,195 @@
-      </dd>
-    </dl>
-
-    <h3 id="_store">Store</h3>
-    <dl>
-      <dt id="stack-components-elasticsearch">
-        Elasticsearch
-      </dt>
-      <dd>
-        Elasticsearch is the distributed search and analytics engine at the heart of the Elastic Stack. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, Elasticsearch can efficiently store and index it in a way that supports fast searches. Elasticsearch provides a REST API that enables you to store data in Elasticsearch and retrieve it. The REST API also provides access to Elasticsearch&#8217;s search and analytics capabilities. <a href="en/elasticsearch/reference/current/elasticsearch-intro.html">Learn more about Elasticsearch</a>.
-      </dd>
-    </dl>
-
-    <h3 id="_consume">Consume</h3>
-    <p>Use Kibana to query and visualize the data that&#8217;s stored in Elasticsearch. Or, use the Elasticsearch clients to access data in Elasticsearch directly from common programming languages.</p>
-    <dl>
-      <dt id="stack-components-kibana">
-        Kibana
-      </dt>
-      <dd>
-        Kibana is the tool to harness your Elasticsearch data and to manage the Elastic Stack. Use it to analyze and visualize the data that&#8217;s stored in Elasticsearch. Kibana is also the home for the Elastic Enterprise Search, Elastic Observability and Elastic Security solutions. <a href="en/kibana/current/introduction.html">Learn more about Kibana</a>.
-      </dd>
-      <dt id="stack-components-elasticsearch-clients">
-        Elasticsearch clients
-      </dt>
-      <dd>
-        The clients provide a convenient mechanism to manage API requests and responses to and from Elasticsearch from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. <a href="en/elasticsearch/client/index.html">Learn more about the Elasticsearch clients</a>.
-      </dd>
-    </dl>
-  </details>
-
-  <h2>Get started</h2>
-  <p>New to Elastic? Learn how you can make the most of your data with the Elastic Stack.
-    Get hands-on with a solution and quickly see data in action, or start from a blank page. </p>
-  <div class="row my-4">
-    <div class="col-md-4 col-12 mb-2">
-      <a class="no-text-decoration" href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
-        <div class="card h-100">
-          <h4 class="mt-3">
-            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt11200907c1c033aa/634d9da119d8652169cf9b2b/enterprise-search-logo-color-32px.png');"></span>
-            Search
-          </h4>
-          <p>Build custom applications with your data using Elasticsearch.</p>
-        </div>
-      </a>
+  <div role="tabpanel" id="solution-tab-contents" aria-labelledby="solution-tab" tabindex="0">
```

Comment:
Can we add **View guide →** links like we have on the serverless landing page? I think it makes it more clear to users that the area is clickable and will send them to the docs.

Reply:
View Elasticsearch docs →
View Observability docs →
View Elastic Security docs →

Reply:
That text currently only exists in the redesigned version of this page, so for now there is no impact here.